src/expression-modifiers/accumulation/properties.js

```diff
@@ -128,7 +128,12 @@ export default function (rootPath, translationKeys = {}) {
         },
       },
     },
-    show(itemData) {
+    show(itemData, handler, args) {
+      const modifierTypes = args.ext.support.modifiers;
+      const supported = modifierTypes && Array.isArray(modifierTypes) && modifierTypes.indexOf(MODIFIER_TYPE) !== -1;
```

Comment:
no need to check both `modifierTypes && Array.isArray(modifierTypes)` - it's enough with `Array.isArray(modifierTypes)`